### DISCUSSION TITLE
mega.nz - remove pop-up upgrade: reached limit

### DISCUSSION THREAD

**Discussion by vwdeiu** [uBlockOrigin/uAssets#20127] [Q&A] [answered]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] This is NOT a [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/19976), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report.
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/124740436/235392297-c4fc6290-73a6-4a70-96a9-3f6a7683f21d.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [X] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://mega.nz/`

### Category

nuisance

### Description

When you;re capapcity is almost full there is a really annoying pop-up that shows everytime you open your account. I tried blocking this with the element picker tool, but there;s a blurry background that can;t be removed.

### Other extensions used

no need.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://github.com/uBlockOrigin/uAssets/assets/38797606/c6a02273-5d57-47e7-bcf3-ecd0520c2c43)
![image](https://github.com/uBlockOrigin/uAssets/assets/38797606/c6650290-fbc3-49e5-9172-6d510650e0cf)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.52.2
Chromium: 118
filterset (summary):
 network: 105947
 cosmetic: 39996
 scriptlet: 18713
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 5-0, never
  easylist: 71861-11, now
  easyprivacy: 33186-64, now
  plowe-0: 3729-1069, now
  ublock-badware: 7271-132, now
  ublock-filters: 34957-145, now
  ublock-privacy: 561-6, now
  ublock-quick-fixes: 226-87, now
  ublock-unbreak: 2124-31, now
  urlhaus-1: 12425-0, now
filterset (user): [array of 5 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 205 ms (selfie)
 maxAssetCacheWait: 122 ms
popupPanel:
 blocked: 0
 extended:
  ##.almost-full.fm-notification-block
  ##.js-lpbtn.btn-link
```
</details>

**Comment by mapx-** [comment, marked as answer]
to the blurry thing add
`:style(filter: none !important)`

see if works

maybe
`mega.nz##.main-blur-block:style(filter: none !important)`
or
`mega.nz##*:style(filter: none !important)`

**Reply by mapx-** [reply]
converted in discussion as such filters wont be added to uBO

**Reply by vwdeiu** [reply]
thaaank you!!!! worked! :)
